studio/components/interfaces/Functions/EdgeFunctionDetails/EdgeFunctionDetails.tsx

```diff
@@ -0,0 +1,322 @@
+              }
+            }, [selectedFunction])
+            return (
+              <>
+                <FormPanel
+                  disabled={!canUpdateEdgeFunction}
+                  footer={
+                    <div className="flex py-4 px-8">
+                      <FormActions
+                        form={formId}
+                        isSubmitting={isSubmitting}
+                        hasChanges={hasChanges}
+                        handleReset={handleReset}
+                        helper={
+                          !canUpdateEdgeFunction
+                            ? 'You need additional permissions to update this function'
+                            : undefined
+                        }
+                      />
+                    </div>
+                  }
+                >
+                  <FormSection header={<FormSectionLabel>Function Details</FormSectionLabel>}>
+                    <FormSectionContent loading={selectedFunction === undefined}>
+                      <Input id="name" name="name" label="Name" />
+                      <Input
+                        disabled
+                        id="slug"
+                        name="slug"
+                        label="Slug"
+                        value={selectedFunction?.slug}
+                      />
+                      <Input
+                        disabled
+                        copy
+                        label="Endpoint URL"
+                        value={functionUrl}
+                        descriptionText="All functions are deployed globally"
+                      />
+                      <Input
+                        disabled
+                        label="Created at"
+                        value={dayjs(selectedFunction?.created_at ?? 0).format(
+                          'dddd, MMMM D, YYYY h:mm A'
+                        )}
+                      />
+                      <Input
+                        disabled
+                        label="Last updated at"
+                        value={dayjs(selectedFunction?.updated_at ?? 0).format(
+                          'dddd, MMMM D, YYYY h:mm A'
+                        )}
+                      />
+                      <Input disabled label="Deployments" value={selectedFunction?.version ?? 0} />
+                    </FormSectionContent>
+                  </FormSection>
+                  <FormSection header={<FormSectionLabel>Function Configuration</FormSectionLabel>}>
+                    <FormSectionContent loading={selectedFunction === undefined}>
+                      <Toggle
+                        id="verifyJwt"
+                        name="verifyJwt"
+                        label="Enforce JWT Verification"
+                        descriptionText="Require a valid JWT in the authorization header when invoking the function"
+                      />
+                      <Toggle
+                        disabled
```

Comment:
For users (like me) who are new to import maps, it could be a little confusing why this is constantly disabled. I'm assuming it's an option that has to be changed via the cli? Could you add a tooltip or a bit of text explaining how to get it enabled?

Reply:
Probably, better to remove the toggle and add a note saying import map enabled / disabled for this function. Also, we can link to this documentation page https://supabase.com/docs/guides/functions/import-maps

Reply:
Good idea. Joshen already has the docs covered under the (?).

<img width="669" alt="Screenshot 2023-03-06 at 5 27 12 pm" src="https://user-images.githubusercontent.com/10985857/223034852-dd8f5d72-c4e6-460f-b6f8-a83a849096c8.png">